### DISCUSSION TITLE
Cannot filter notifications/issues by author:dependabot. Bug? Or am I doing something wrong?

### DISCUSSION THREAD

**Discussion by mcclure** [community/community#40855] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If I go into my notifications, and I type in the filter bar "author:shadowfacts", it will show me all issues/PRs created by user "shadowfacts".

![image](https://user-images.githubusercontent.com/277318/205702334-aa2e6032-fb15-44c7-8ddc-df049dbbc291.png)

However if I type in the filter bar "author:dependabot", it shows me nothing.

![image](https://user-images.githubusercontent.com/277318/205702400-f12b5b53-3c11-4d0e-afe0-23e39a4dea1b.png)

Despite this I have [a number of PRs](https://github.com/mcclure/ts-hello-bug/pull/9) in my notifications from a user named "Dependabot".

Incidentally, if I go to the PR page of the project linked just there above, it appears I can't search for "author:dependabot" in *any* view:

![image](https://user-images.githubusercontent.com/277318/205703445-d8acaf0a-ebaa-44d2-a37c-b5beb7d7bcf8.png)

If I click Dependabot's name, it does not take me to a normal user page. It takes me to [this Github help page](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates). This suggests to me "Dependabot" may be something special in the Github database which is "not exactly a user", and therefore there may be a bug or code oversight preventing the author: filter from working with it.

Regardless, it would be very useful to me to filter by author:dependabot (for the same reason it is useful to filter by any other author).

Is this a bug or am I doing something wrong? Is this the correct place to report a bug?

**Comment by airtower-luna** [comment]
Taking a hunch from the author name in the commit, `author:dependabot[bot]` works for me. That could definitely be a lot more obvious. :sweat_smile:

**Comment by mcclure** [comment]
Interesting! While I am glad I have the answer, I think Github should still treat this as a "UX bug" because that solution is not very discoverable.

On a related note, it seems a nice extension would be if searching author:[bot] could match any bot. But I don't know of any bots other than dependabot.
